examples/user_interfaces/embedding_in_tk_sgskip.py

```diff
@@ -21,7 +21,10 @@
 
 fig = Figure(figsize=(5, 4), dpi=100)
 t = np.arange(0, 3, .01)
-fig.add_subplot().plot(t, 2 * np.sin(2 * np.pi * t))
+ax = fig.add_subplot()
+line = ax.plot(t, 2 * np.sin(2 * np.pi * t))
```

Comment:
We usually do:
```suggestion
line, = ax.plot(t, 2 * np.sin(2 * np.pi * t))
```
to unpack it directly. And then it'll match the name.